core/lib/generators/spree/mailers_preview/templates/mailers/previews/order_preview.rb

```diff
@@ -6,4 +6,8 @@ def confirm_email
   def cancel_email
     Spree::OrderMailer.cancel_email(Spree::Order.complete.first)
   end
+
+  def store_owner_notification_email()
```

Comment:
if you don't pass arguments, you can ommit `()`

Reply:
yes, thanks. I've noticed this already. It was added by accident.